src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyController.cs

```diff
@@ -137,6 +137,49 @@ public async Task<ViewResult> ChooseOrganisation()
         {
             return View(new ChooseOrganisationViewModel());
         }
+
+        [HttpGet]
+        [Route("need-bank-details")]
+        public async Task<ViewResult> BankDetailsConfirmation()
+        {
+            return View(new BankDetailsConfirmationViewModel());
+        }
+
+        [HttpPost]
+        [Route("need-bank-details")]
+        public async Task<IActionResult> BankDetailsConfirmation(string hashedAccountId, BankDetailsConfirmationViewModel viewModel)
```

Comment:
One to discuss but you could put the accountId on the view model and it will bind to it so the method only has one parameter.

Reply:
Added account id and legal entity id to the model as we may need both